src/main/java/edu/byu/ece/rapidSmith/design/subsite/CellPin.java

```diff
@@ -397,4 +397,9 @@ public String toString() {
 	 * @return 
 	 */
 	public abstract CellPin getExternalPin();
+
+	public void setMacroPinToGlobalNet(CellNet n) {
+		setNet(n);
```

Comment:
Why create this method instead of just using setNet()?

Reply:
setNet() is not public and I assumed normal (non-careful) usage of it would corrupt the data structures.  This one is public and hopefully the name will suggest its intent for limited usage.